auxiliaries/pipeline_auxiliaries.py

```diff
@@ -457,6 +457,17 @@ def get_cluster_size_from_name(path):
     return float(os.path.splitext(path)[0].split('clusterSize_')[1])
 
 
+def get_number_samples_from_name(path, biological_condition):
+    return os.path.splitext(path)[0].split(f'{biological_condition}_')[1].split('_')[0]
+
+
+def get_unique_members_from_name(path):
+    unique_members = os.path.splitext(path)[0].split('uniqueMembers_')[1].split('_')[0]
+    if 'top' in unique_members:

```

Comment:
Also, please provide an example. When does top appear?

Reply:
I made mistake and there is no top in this clusters.